src/views/docs-view/utils/get-deploy-preview-loader.ts

```diff
@@ -104,8 +104,8 @@ export function getDeployPreviewLoader({
 
 			return [
 				...remarkTerraformPlugins,
-				...remarkPluginsFromExtraOptions,
 				...remarkPluginsForFileSystemContent,
+				...remarkPluginsFromExtraOptions,
```

Comment:
put these after `remarkPluginsForFileSystemContent` ensures that partials are resolved before we do the link rewrites.

Reply:
I think this is a worthy code comment!

Reply:
Upd8ed